stories/datafusion.stories.js

```diff
@@ -79,16 +79,21 @@ export const getMapProcessing = () => {
         //VERSION=3
         var setup = () => ({
           input: [
-            {datasource: "l2a", bands:["B02", "B03", "B04"], units: "REFLECTANCE", mosaicking: "ORBIT"},
-            {datasource: "l1c", bands:["B02", "B03", "B04"], units: "REFLECTANCE"}],
+            {datasource: "s2l2a", bands:["B02", "B03", "B04"], units: "REFLECTANCE", mosaicking: "ORBIT"},
```

Comment:
Extreme nitpicking: title of  story should be changed to `Data fusion - Sentinel-2 L1C and Sentinel-2 L2A` as Landsat 8 is not used in this example. fftr

Reply:
Thank you, fixed!